.github/workflows/publish-cli.yml

```diff
@@ -182,6 +210,6 @@ jobs:
           make -- --dist-homebrew
 
       - name: Install via Homebrew
-        if: steps.github-release.outputs.release-url != ''
+        if: steps.github-release.outputs.release-url != '' && steps.check-homebrew.outputs.needs-publish == 'true'
         run: |
           make install-brew
```

Comment:
maybe it's out of this PR, but probably we need to test if the actual version of `vespa-cli` is installed via brew?